examples/jsm/renderers/webgl/nodes/GLSLNodeBuilder.js

```diff
@@ -253,6 +255,30 @@ ${ flowData.code }
 
 	}
 
+	getTypeFromAttribute( attribute ) {
+
+		let nodeType = super.getTypeFromAttribute( attribute );
+
+		if ( /^[iu]/.test( nodeType ) && attribute.gpuType !== IntType ) {
+
+			let dataAttribute = attribute;
+
+			if ( attribute.isInterleavedBufferAttribute ) dataAttribute = attribute.data;
+
+			const array = dataAttribute.array;
+
+			if ( ( array instanceof Uint32Array || array instanceof Int32Array ) === false ) {
+
+				nodeType = nodeType.slice( 1 );
```

Comment:
Something like `this.getTypeFromLength( this.getTypeLength( nodeType ) )` may be more reliable? /ping @sunag

Reply:
In this specific case I think it is not necessary because of the conditional: `/^[iu]/.test( nodeType )`